### PR TITLE
Refactor Druid-Single-Container Image

### PR DESCRIPTION
**Pull request by Konstantinos-Chaitas-PubNative**
After the initial refactoring in Druid that took place [here](https://github.com/pubnative/docker-images/pull/200), I noticed that the following things can be improved in the `single-container-druid` image building:

- Remove duplicate and unnecessary Makefile, use only a single one
- Simplify and cleanup Dockerfile
- Fix security hole with SHA
- Update README following the same convention as the parent project

The new image has be created and pushed [here](https://hub.docker.com/layers/pubnative/single-container-druid/v28.0.0-21f6b08/images/sha256-a721e1660e7c99100f26418485e51ec1631b464ce54985c03fe4b434d7734a9a?context=repo). **No changes in functionality are expected, everything should work exactly as before**

I confirm that I have added:

- [X] A comment to every Dockerfile with information about target repository and tag (version).
- [X] Filled in readme on dockerhub and in this repo about how to build the image if some extra steps should be done.

### CODE REVIEW

File: druid/single-container-druid/Dockerfile [hunk 1]
```diff
@@ -8,24 +6,19 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 ARG DRUID_VERSION
+ARG DRUID_JAR=apache-druid-${DRUID_VERSION}-bin.tar.gz
+ARG DRUID_URL=https://downloads.apache.org/druid/${DRUID_VERSION}/${DRUID_JAR}
+ARG DRUID_SHA=c971ccbb4220fa2ae9f8a0a0f82c3e4b87750d27ac15d863d1e9f415863ccb8b8e5c764122dc8ef76044a42d27e1108b980ca2b88de8aff660db5a22356d6ab7
```
Comment (alifouzizadeh) [inline druid/single-container-druid/Dockerfile:11]: In this version the SHA is hardcoded, but previously we were downloading the version and the SHA  and then compering them. Shall we use the previous approach ?

Comment (alifouzizadeh) [reply druid/single-container-druid/Dockerfile:11]: Where did you get the `SHA` ?

Comment (Konstantinos-Chaitas-PubNative) [reply druid/single-container-druid/Dockerfile:11]: Same URL you were using, there is only one URL in fact

Comment (alifouzizadeh) [reply druid/single-container-druid/Dockerfile:11]: But in this case you need to download the SHA every time we change the Druid version from the same sources, right?

Comment (Konstantinos-Chaitas-PubNative) [reply druid/single-container-druid/Dockerfile:11]: Yes, that's the point of SHA, otherwise it is literally useless :)

Comment (alifouzizadeh) [reply druid/single-container-druid/Dockerfile:11]: If the two files were in two different sources I'm totally with you, but they are in the same source.I was downloading the two files at the same time: 
```
 RUN wget -q -O /tmp/druid.tar.gz ${DRUID_DOWNLOAD_URL} \
    && wget -q -O /tmp/druid.tar.gz.sha512 ${DRUID_SHA_URL}
```
So, I trusted the source one time as you did :)). But mine was two files that is the difference. I just wanted to keep this automatic as much as possible, that was the reason behind my decision. But this one works for me.

**Review by alifouzizadeh — APPROVED** [review]

**Pull request merged**